src/FixerDefinition/VersionSpecification.php

```diff
@@ -36,19 +36,21 @@
     public function __construct($minimum = null, $maximum = null)
     {
         if (null === $minimum && null === $maximum) {
-            throw new \InvalidArgumentException('Either minimum or maximum need to be specified');
+            throw new \InvalidArgumentException('Minimum and/or maximum need to be specified.');
```

Comment:
I would use _and/or_ too but I often use the equivalent in french as well so not sure it's actually valid in english :)